src/olympia/users/forms.py

```diff
@@ -395,6 +396,14 @@ def __init__(self, *args, **kwargs):
             if not self.instance.is_developer:
                 choices = email.NOTIFICATIONS_CHOICES_NOT_DEV
 
+            if self.instance.fxa_migrated():
+                self.fields['email'].required = False
+                self.fields['email'].widget = forms.EmailInput(
+                    attrs={'disabled': 'disabled'})
```

Comment:
I think `disabled` makes more sense other than the text selection thing. It was easy enough to change though so I made it `readonly`.

FWIW not being able to select the text of a `disabled` input seems to be a quirk of Firefox. Chrome and Safari both allow it.